integration-tests/suites/config/images.go

```diff
@@ -0,0 +1,70 @@
+package config
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+type ImageStore struct {
+	Images map[string]string
+}
+
+func (i *ImageStore) CollectorImage() string {
+	return ReadEnvVar(envCollectorImage)
+}
+
+func (i *ImageStore) GRPCImage() string {
```

Comment:
I think I understand that the GRPC mock is so much used everywhere that it deserves its own method. My first guts feeling was that every image could have a dedicated method; the advantage being that the compiler can check for typos in method names whereas it cannot with keys (and also a source browser can give you all usage locations).

But after more thinking, a string-search is rather effective, the tests will spot typos and you might have found just the right balance.

Reply:
That is a very good point, it's just as static as the other images (I think Collector is the only one that's dynamic?) I'll remove this method and just get the image in the same way as all the others